apolo-cli/src/apolo_cli/formatters/utils.py

```diff
@@ -153,3 +153,38 @@ def yes() -> str:
 
 def no() -> str:
     return "[red]×[/red]"
+
+
+def format_multiple_gpus(entity: Union[_NodePool, Preset], sep: str = "\n") -> str:
```

Comment:
Parameter `sep` is never passed, please drop it and use a constant instead

Reply:
@asvetlov changed this to a const